client/README.md

```diff
@@ -1,272 +1,72 @@
-present in the archive. Until the package is not published to
-crates.io, we need to generate it ourselves:
-
-```bash
-./debian/generate_checksums.py
-```
-
-Then, build the source package:
-
-```bash
-dpkg-buildpackage -S #-k=<key-to-sign> if you have more than one GPG key for the specified DEBEMAIL
-```
-
-You can also `lintian --pedantic` to statically check the files under
-the `debian/` dir.
-
-### Testing the package build
-
-You can test your package and build it with the
-[sbuild](https://wiki.debian.org/sbuild) tool. In this example, we do
-it for questing distro, but you can replace it with the desired one:
-
-```bash
-sudo apt install sbuild mmdebstrap uidmap
-mkdir -p ~/.cache/sbuild
-mmdebstrap --variant=buildd --components=main,restricted,universe questing ~/.cache/sbuild/questing-amd64.tar.zst
-```
-
-For configuring `sbuild` , install `sbuild-debian-developer-setup`:
-
-```bash
-sudo apt install sbuild-debian-developer-setup
-sudo sbuild-debian-developer-setup
-newgrp sbuild
-```
-
-Update chroot:
-
-```bash
-sudo sbuild-update -udcar u
-schroot -l | grep sbuild
-```
-
-Put the following content into `~/.sbuildrc` file:
-
-```perl
-$chroot_mode = 'unshare';
-$run_autopkgtest = 0;
-$autopkgtest_root_args = '';
-$autopkgtest_opts = [ '--apt-upgrade', '--', 'unshare', '--release', '%r', '--arch', '%a' ];
-```
-
-Not you can build the binary itself:
-
-```bash
-sbuild /path/to/.dsc -d questing
-```
-
-### Running autopkgtests locally in VM
-
-Before proceeding please make sure you have QEMU installed on your
-system.
-
-To run autopkgtests, first we need to download the image (replace
-`questing` with a corresponding release):
-
-```sh
-autopkgtest-buildvm-ubuntu-cloud -r questing -v \
- --cloud-image-url http://cloud-images.ubuntu.com/daily/server
-```
-
-The image size may be too small, so you probably need to resize the
-disk and give it more storage:
-
-```
-qemu-img resize autopkgtest-questing-amd64.img +15G
-```
-
-Then run the autopkgtests. The setup command adds more space to the
-`tmpfs` partition because autopkgtest for cargo uses the `/tmp`
-directory for building the package.
-
-```sh
-autopkgtest \
-  --apt-upgrade \
-  --shell-fail \
-  --output-dir dep8-rust-hwlib \
-  --setup-commands="mount -o remount,size=10G /tmp" \
-    /path/to/<package>_<version>_source.changes \
-  -- qemu --ram-size 4096 /var/lib/adt-images/autopkgtest-questing-amd64.img
-```
+- `hwlib` is released under the [LGPL-3.0 license](./hwlib/LICENSE).
+- `hwctl` is released under the [GPL-3.0 license](./hwctl/LICENSE).
 
-### Publishing the package
+© 2025 Canonical Ltd.
 
-After the archive is created and you've tested the build locally, you
-can publish the package by running:
+[^sudo]:
+    The client requires root access since we collect the hardware information
+    using SMBIOS data. If you're running it on a device that doesn't have SMBIOS
+    data available, root privileges are not required.
```

Comment:
Technically we currently support only amd64 architecture so if SMBIOS is not available, the tool is not runnable